csharp-advanced-topics/ActionFuncDelegatesCsharp/ActionFuncDelegatesCsharp/UnderstandingDelegates.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace ActionFuncDelegatesCsharp
+{
+    public class UnderstandingDelegates
+    {
+        /// <summary>
+        /// Returns full name concatenating first name and last name
+        /// </summary>
+        /// <param name="firstName"></param>
+        /// <param name="lastName"></param>
+        /// <returns></returns>
```

Comment:
We rarely use comments, only when necessary. Documentation type comments even less. The examples should be clear enough to the user.